main.tf

```diff
@@ -166,15 +172,12 @@ resource "aws_db_subnet_group" "main" {
   description = "Group of DB subnets"
   subnet_ids  = ["${var.subnets}"]
 
-  tags {
-    envname = "${var.envname}"
-    envtype = "${var.envtype}"
-  }
+  tags = "${merge(var.tags, map("Name", format("%s", var.identifier_prefix)))}"
```

Comment:
You are forcing the use of a Name tag assigned to the identifier_prefix, as well as any tags in the map. Can we not have this?